src/http/params.cr

```diff
@@ -143,7 +143,7 @@ module HTTP
     # params["non_existent_param"] # KeyError
     # ```
     def [](name)
-      raw_params[name].first
+      fetch(name) { raise KeyError.new "Missing param name: #{name.inspect}" }
```

Comment:
If you want to introduce an Exception message please change it to: `Missing key: #{name.inspect}`

Reply:
Sorry, you're right.